### ISSUE TITLE
Cannot load model

### ISSUE THREAD

**Issue by maximus3219** [SaschaDittmann/tfjs-customvision#10]
How come I cannot load the model, when I simply replace with the model generated from my python script with tensorflow?

**Comment by SaschaDittmann** [comment]
TensorFlow.JS uses a different file format. Please check this link to convert your TF model to TFJS https://www.tensorflow.org/js/guide/conversion

**Comment by maximus3219** [comment]
Of course I have converted the TF model to TFJS.
I have tried model based on VGG16, InceptionX3 and EfficientNetB0, but still cannot load.
Can TFJS support these complex model?

One of the errors say:
jquery-3.3.1.slim.min.js:2 Uncaught Error: Unknown layer: RandomFlip. This may be due to one of the following reasons:
1. The layer is defined in Python, in which case it needs to be ported to TensorFlow.js or your JavaScript code.
2. The custom layer is defined in JavaScript, but is not registered properly with tf.serialization.registerClass().
    at jN (generic_utils.js:242)
    at GI (serialization.js:31)
    at e.fromConfig (models.js:1026)
    at jN (generic_utils.js:277)
    at GI (serialization.js:31)
    at e.fromConfig (models.js:1026)
    at jN (generic_utils.js:277)
    at GI (serialization.js:31)
    at models.js:295
    at u (runtime.js:45)

What could be the problem?
